src/OpenGL/MonoMacGameView.cs

```diff
@@ -120,24 +122,36 @@ public override void DrawRect (CGRect dirtyRect)
 		public override void LockFocus ()
 		{
 			base.LockFocus ();
-			if (openGLContext.View != this)
-				openGLContext.View = this;
+			if (ActualOpenGLContext.View != this)
+				ActualOpenGLContext.View = this;
 		}
 
-		public NSOpenGLContext OpenGLContext {
-			get { return openGLContext; }
+		public NSOpenGLContext? OpenGLContext {
+			get => openGLContext;
+		}
+
+		public NSOpenGLContext ActualOpenGLContext {
+			get {
+				if (openGLContext is null)
+					throw new InvalidOperationException ("Operation requires an OpenGLContext, which hasn't been created yet.");
+				return openGLContext;
+			}
 		}
 
 		public NSOpenGLPixelFormat PixelFormat {
-			get { return pixelFormat; }
+			get {
+				if (pixelFormat is null)
+					throw new InvalidOperationException ("Operation requires a PixelFormat that cannot be null.");
+				return pixelFormat;
```

Comment:
This property has AFAIK the same issue as the OpenGL one we talked about. You are throwing an exception on a property that did not used to and returned a null variable correctly. We are changing the behaviour and we should not.